lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -5,10 +5,25 @@
 import type { APICallInfo } from './config';
 
 /**
- * Object representing a file, folder or document stored by the LMS
+ * MIME types which are recognized by the frontend.
+ *
+ * These are used to choose appropriate icons to represent the file.
+ */
+export type MimeType = 'text/html' | 'application/pdf' | 'video';
```

Comment:
Note that `video` here is a "top-level" MIME type that can represent any video type (mp4, webm etc.). Canvas Studio videos all use `.mp4` from what I've seen.